rilldata/rill-developer
# Tighten up Onboarding screen

## Checklist

## Summary
#### Issue addressed: 
The onboarding screen (at the root page `/`) has confused some people who mistook each list item for a clickable button. Additionally, the copy is outdated now that we've changed the Add Source button (#2538).

#### Details:
Janet did a light design pass over the screen. [Here's the Figma design](https://www.figma.com/file/ADt3OivNV54xeoXkUpPxP2/RILL?type=design&node-id=1685%3A35815&t=2eEg72rm9Bo4Z88J-1). This PR implements this design.

This PR does _not_ create a new `secondary` button style. Hence the button does not match the mock exactly. Changing the `secondary` button can be done in a separate PR that considers the `secondary` buttons across the application.

## Steps to Verify
1. Open the onboarding screen
2. Check that it matches the Figma mock
3. Check that the Add Data button works correctly
